dipy/denoise/noise_estimate.py

```diff
@@ -136,7 +139,8 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
     return sigma
 
 
-def _piesno_3D(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
+@deprecated_params("l", "step", since="1.9", until="2.0")
```

Comment:
`since="1.10.0", until="1.12.0"` instead

Reply:
Done in 7b16a78.